components/Article/ArticlePage.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import Head from 'next/head';
+import 'antd/dist/antd.css';
+import { Pagination } from 'antd';
+import NavigationBar from '../Common/NavigationBar';
+import { REPORT } from './constants';
+import { HOME_STRINGS } from '../constants';
```

Comment:
keep all the constants for this component in the same place `import { HOME_STRINGS, REPORT  } from './constants';`